plugin/mouse/mouse.py

```diff
@@ -132,15 +132,14 @@ def mouse_wake():
     def mouse_drag(button: int):
         """Press and hold/release a specific mouse button for dragging"""
         # Clear any existing drags
-        self.mouse_drag_end()
```

Comment:
I was surprised this even worked. We should definitely call Talon actions through the action interface.